test/test_config.rb

```diff
@@ -58,4 +54,8 @@ def check_context(context, ssl: true)
       assert_equal(OpenSSL::SSL::VERIFY_NONE, context.ssl_options[:verify_ssl])
     end
   end
+
+  def config_file(name)
+    File.new(File.join(File.dirname(__FILE__), 'config', name))
+  end
```

Comment:
:+1: for this change, but things usually have a better change of being review if they are as restricted as possible.